data/campus_data/printTypicalDemand_0.csv

```diff
@@ -0,0 +1,8761 @@
+time,kw
```

Comment:
The naming of this file seems arbitrary. Did you generate this with RAVEN and the printTypicalDemand.xml file? What is the meaning of 0 in this context? Is this for the 0th timestep or something?